content/backstage/plugins/github-pull-requests.md

```diff
@@ -62,3 +64,15 @@ gettingStarted:
           </Grid>
         );
 ---
+
+
+## Repository migration notice
```

Comment:
Without a date associated with this text, it's impossible to know if this migration is happening today, if it's scheduled for a month from now, or if it actually already happened and we forgot to take down the notice.

```suggestion
## Repository migration notice (June/July 2021)
```

Also we need a reminder system to make sure we don't forget to take these notices down at the appropriate time.

Reply:
@dtuite Agree, we can keep it non-merged until we publish the new version and make sure it works